### PR TITLE
added Add/Delete/Update functionality

### PR DESCRIPTION
**Pull request by omjadhav85**

### CODE REVIEW

File: src/utils/taskActions.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import { v4 as uuid } from "uuid";
+import { defaultNewTask } from "../components/Modal/defaultNewTask";
+
+export const validateTask = (newTask, setError) => {
+  const { taskTitle, taskDescription, taskTime } = newTask;
+
+  if (taskTitle.length === 0) {
+    setError("Title cannot be blank!");
```
Comment (ajay-mandaviya) [inline src/utils/taskActions.js:7]: if taskTitle is your string , 
then you don't need to  use length 
you can use condition like
 if(!taskTitle){ 
setError("Title cannot be blank!")
}

File: src/utils/taskActions.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    return false;
+  }
+
+  if (taskDescription.length === 0) {
+    setError("Description cannot be blank!");
+    return false;
+  }
+
```
Comment (ajay-mandaviya) [inline src/utils/taskActions.js:15]: same here

**Review by ajay-mandaviya — APPROVED** [review]

**Review by junaidshaikh-js — APPROVED** [review]
Good work done! 

Nice folder structure and clean code. Changes suggested by fellow reviewer can be done.

File: src/utils/taskActions.js (hunk 1 above)
Comment (omjadhav85) [reply src/utils/taskActions.js:7]: Hey thanks for the suggestion. Made the changes

**Comment by omjadhav85** [comment]
Thanks Ajay and Junaid! Merging PR after 2 reviews

**Pull request merged**